_src/dashboard.md

```diff
@@ -9,10 +9,23 @@ nav: Dashboard
     <div class="graphic" id="chart-daily-positive-total"></div>
     <div class="graphic" id="chart-daily-death-total"></div>
   </div>
+  <br />
+  <div class="map" title="State Positive Cases / million people" id="state-map">
 </div>
 
-
 <link rel="stylesheet" href="/_assets/css/c3.css" >
+ <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+   integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+   crossorigin=""/>
+
+<link rel="stylesheet" href="/_assets/css/_dashboard.css" >
 <script src="/_assets/js/d3.js"></script>
 <script src="/_assets/js/britecharts.js"></script>
-<script src="/_assets/js/dashboard-charts.js"></script>
\ No newline at end of file
+
+ <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
```

Comment:
Can you fix your tag alignments here. They seem to be on different tab/space levels.

Reply:
done